src/OSPSuite.Presentation/Presenters/SimulationPredictedVsObservedChartPresenter.cs

```diff
@@ -112,6 +116,16 @@ private void plotCalculationColumn(DataColumn calculationColumn, Action<DataColu
          });
       }
 
+      private void addDeviationLines(float foldValue)
+      {
+         var observationColumns = getAllAvailableObservedData().Select(x => x.FirstDataColumn()).ToList();
+         _deviationLineRepositories.AddRange(_predictedVsObservedChartService.AddDeviationLine(foldValue, observationColumns, Chart, _deviationLineRepositories.Count));
```

Comment:
and here it's not devided by 2

Plus this code is so much the same as the other one,. I think we need to compose here

Reply:
About the code duplication. Both call the service to get the work done, the main difference I would say is getting the observation columns....